packages/react-data-grid/src/Grid.tsx

```diff
@@ -89,36 +93,39 @@ export default function Grid<R>({ emptyRowsView, headerRows, ...props }: GridPro
           {createElement(emptyRowsView)}
         </div>
       ) : (
-        <Viewport<R>
-          rowKey={props.rowKey}
-          rowHeight={props.rowHeight}
-          rowRenderer={props.rowRenderer}
-          rowGetter={props.rowGetter}
-          rowsCount={props.rowsCount}
-          selectedRows={props.selectedRows}
-          columnMetrics={props.columnMetrics}
-          onScroll={onScroll}
-          cellMetaData={props.cellMetaData}
-          rowOffsetHeight={props.rowOffsetHeight}
-          minHeight={props.minHeight}
-          scrollToRowIndex={props.scrollToRowIndex}
-          contextMenu={props.contextMenu}
-          rowSelection={props.rowSelection}
-          getSubRowDetails={props.getSubRowDetails}
-          rowGroupRenderer={props.rowGroupRenderer}
-          enableCellSelect={props.enableCellSelect}
-          enableCellAutoFocus={props.enableCellAutoFocus}
-          cellNavigationMode={props.cellNavigationMode}
-          eventBus={props.eventBus}
-          interactionMasksMetaData={props.interactionMasksMetaData}
-          RowsContainer={props.RowsContainer}
-          editorPortalTarget={props.editorPortalTarget}
-          overscanRowCount={props.overscanRowCount}
-          overscanColumnCount={props.overscanColumnCount}
-          enableIsScrolling={props.enableIsScrolling}
-          onViewportKeydown={props.onViewportKeydown}
-          onViewportKeyup={props.onViewportKeyup}
-        />
+        grid.current && (
```

Comment:
Why do this?
Since updating refs doesn't trigger a new render, I fear this might lead to situation where the viewport doesn't get rendered.

Reply:
Changed it to ignore the first render only